src/utilities/features/types.ts

```diff
@@ -3,6 +3,5 @@ export interface FeaturesConfig {
 }
 
 export interface Features {
-  newDesignLanguage: boolean;
   [key: string]: boolean;
```

Comment:
TODO: Remove this too so that people who use the newDesignLanguage prop will be alerted that it has gone away.

Features should be an empty object: `export interface Features {}`

Which isn't very useful now but let's keep it around in case we want to add toggles in the future